### PR TITLE
Sign up tests

### PR DESCRIPTION
**Pull request by miguelhenaoa**
## Summary

- Refactor to SignUp component and create SignUpForm component
- Create mock users
- Add unit test for some cases
## Screenshots
image

## Trello Card

https://trello.com/c/DCMTYFjk

### CODE REVIEW

File: src/components/SignUpForm/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+import SignUpForm from '.';
+
+const MAX_REQUIRED_MESSAGES = 5;
+
+const onSubmit = jest.fn();
+
+describe('', () => {
+  beforeEach(() => {
+    // i18n.init();
```
Comment (TaloMontes) [inline src/components/SignUpForm/index.test.tsx:15]: delete comment

Comment (miguelhenaoa) [reply src/components/SignUpForm/index.test.tsx]: Done!

File: src/components/SignUpForm/index.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+          validate: {
+            match: v => v === getValues().password || t('Validations:matchPassword')
+          }
+        })}
+        label={t('SignUp:passwordConfirmation')}
+        type="password"
+        error={errors.passwordConfirmation?.message}
+        idField="passwordConfirmation"
+      />
+
```
Comment (alejobh) [inline src/components/SignUpForm/index.tsx:79]: delete this line

File: src/components/Wrapper/index.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React, { ReactNode } from 'react';
+
+import styles from './styles.module.scss';
+
+interface Props {
+  children: ReactNode;
+}
+
+function Wrapper({ children }: Props) {
+  return <div className={styles.wrapper}>{children}</div>;
```
Comment (alejobh) [inline src/components/Wrapper/index.tsx:10]: this should render `<FormCard>` too

File: src/components/Wrapper/styles.module.scss [hunk 4]
```diff
@@ -0,0 +1,17 @@
+@import "../../scss/variables/colors";
```
Comment (alejobh) [inline src/components/Wrapper/styles.module.scss:1]: does this work? `@import "variables/colors";`

File: src/components/SignUpForm/index.tsx [hunk 5]
```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+
+import { User } from 'utils/types';
+import { regexEmail, regexPassword } from 'constants/forms';
+
+import Input from '../Input';
+
+type Props = {
```
Comment (alejobh) [inline src/components/SignUpForm/index.tsx:10]: this should be an interface

File: src/components/SignUpForm/index.tsx [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+  return (
+    <form onSubmit={handleSubmit(onSubmit)}>
+      <Input
+        {...register('firstName', { required: t('Validations:required') })}
+        label={t('SignUp:firstName')}
+        type="text"
+        error={errors.firstName?.message}
+        idField="firstName"
+      />
+
```
Comment (alejobh) [inline src/components/SignUpForm/index.tsx:33]: delete line

**Review by alejobh — CHANGES_REQUESTED** [review]

**Review by alejobh — APPROVED** [review]

**Review by alejobh — APPROVED** [review]

**Pull request merged**